src/settings/components/utils.ts

```diff
@@ -496,13 +496,32 @@ export function defaultAsFirstOrderIdentities(identities: Array<Identity>): Arra
 	return result;
 }
 
+/**
+ * Return the requested attribute value as an array if present, or an empty array otherwise.
+ *
+ * @param settings
+ * @param key
+ * @returns
+ */
+export function getAttributeIfPresent(settings: AccountSettings, key: string): Array<string> {
+	const attributeValue = settings.attrs[key];
+	if (attributeValue) {
+		if (isArray(attributeValue)) {
```

Comment:
prefer `Array.isArray` instead of lodash (we are trying to remove lodash, so in almost every refactor we try to replace it with pure js utils)